### PR TITLE
Number and fraction params for select_index_values

### PR DESCRIPTION
**Pull request by ianspektor**

### CODE REVIEW

File: temporian/core/operators/select_index_values.py [hunk 1]
```diff
@@ -36,14 +36,38 @@
...
+
+        if keys is not None:
+            normalized_keys = normalize_index_key_list(keys)
+            self._keys = normalized_keys
+            self.add_attribute("keys", normalized_keys)
+        else:
+            self._keys = None
+
+        self._number = number
+        if number is not None:
```
Comment (achoum) [inline temporian/core/operators/select_index_values.py:63]: Nit: Check that "number" is positive.

File: temporian/implementation/numpy/operators/select_index_values.py [hunk 2]
```diff
@@ -35,10 +36,26 @@ def __call__(self, input: EventSet) -> Dict[str, EventSet]:
...
         # Create output EventSet
         output_evset = EventSet(data={}, schema=output_schema)
 
+        if number is not None:
+            if number > len(all_keys):
+                number = len(all_keys)
+            keys = random.sample(all_keys, k=number)
+
+        elif fraction is not None:
+            keys = random.sample(all_keys, k=int(len(all_keys) * fraction))
```
Comment (achoum) [inline temporian/implementation/numpy/operators/select_index_values.py:53]: Round or ceil?

**Review by achoum — APPROVED** [review]
Awesome

File: temporian/implementation/numpy/operators/select_index_values.py (hunk 2 above)
Comment (ianspektor) [reply temporian/implementation/numpy/operators/select_index_values.py:54]: rounds, added note in code and docstring

**Pull request merged**